### ISSUE TITLE
Batchsize and lr of HTC+Swin-L

### ISSUE THREAD

**Issue by 962858249** [microsoft/SoftTeacher#75]
Thank you for your outstanding work, I only have 8 A-100 cards, can I train a batchsize each card and adjust the learning rate to 0.0001 / 2? Is there a negative impact on this relatively large model accuracy of HTC+swin-L?
